src/data/corpus.py

```diff
@@ -0,0 +1,46 @@
+import re
+
+document = str(input("Please insert the full path to the text (without quotation marks): "))
+
+f = open(document, "r", encoding="utf-8")
+text = f.read()
+
+
+def un_gutenberg(text):
+    """Removes the Project Gutenberg intro and outro"""
+    new_text = re.search("[*][*][*] [a-zA-ZÀ-ú.:;,!-? ]+ [*][*][*]", text)
+    no_intro = text[new_text.end():].strip()
+
+    new_text = re.search("[*][*][*] [a-zA-ZÀ-ú.:;,!-? ]+ [*][*][*]", no_intro)
+    no_outro = no_intro[:new_text.start()].strip()
+
+    return no_outro
+
+
+beginnings = ["Chapitre I\n", "CHAPITRE I\n", "I\n", "LIVRE PREMIER\n", "Livre premier\n"]
+endings = ["NOTES", "NOTE", "Notes"]
+
+
+def clean_up(text):
+    """Tries to clean up the text, removing the beginning notes,
+    the ending notes and the unnecessary characters in between"""
+    for item in beginnings:
+        if item in text:
+            new_doc = text[text.find(item):]
+    for item in endings:
+        if item in new_doc:
+            new_doc = new_doc[:new_doc.find(item)]
+    cleaned_new_doc = re.sub("--", "—", new_doc) #substitutes -- for — in direct speech
+    cleaned_new_doc = re.sub("_", "", cleaned_new_doc) #cleans up strings like _ANANKÊ_
+    cleaned_new_doc = re.sub(' +', ' ', cleaned_new_doc) #cleans up redundant spaces
+    cleaned_new_doc = re.sub('(\[.*\])', "", cleaned_new_doc) #cleans up things like [1]
+    cleaned_new_doc = re.sub("[. ]+[.]", "", cleaned_new_doc) #cleans up . . .
+    cleaned_new_doc = re.sub('[* ]+[*]', "", cleaned_new_doc) #cleans up * * *
+    cleaned_new_doc = re.sub('\n\n+', "\n", cleaned_new_doc) #cleans up leftover blank lines
+    return cleaned_new_doc
+
+
+if "Gutenberg" in text:
+    text = un_gutenberg(text)
+
+print(clean_up(text))
```

Comment:
i guess at the end we wanna save it somewhere, not just print

Reply:
btw, since your added data is raw, we'll put in in `__data/row` folder
processed data will then be in `__data/silver`, and the fully prepared, ready-to-use data will be in `__data/gold`

Reply:
I don't know how to save it... Help would be appreciated